testsuite/src/test/java/org/jboss/intersmash/tests/wildfly/message/broker/amq/AmqBrokerSslApplication.java

```diff
@@ -0,0 +1,448 @@
+	 */
+	public static int getMessagesCount(String queueName) {
+		int messagesCount = 0;
+		Map<String, Map<String, Integer>> podInfo = new HashMap<>();
+		// get broker pods: they are labeled with the CRD kind they were created from
+		List<Pod> pods = OpenShifts.master().getLabeledPods(ActiveMQArtemis.class.getSimpleName(), NAME);
+		for (Pod pod : pods) {
+			Map<String, Integer> queueInfo = new HashMap<>();
+			podInfo.put(pod.getMetadata().getName(), queueInfo);
+			// get queues on the broker pod
+			String cmdArtemis = String.format(
+					"/home/jboss/amq-broker/bin/artemis queue stat --user=%s --password='%s' --json --url=tcp://%s:61616",
+					ADMIN_USER,
+					ADMIN_PASSWORD,
+					pod.getStatus().getPodIP());
+			PodShellOutput queues = OpenShifts.master().podShell(pod).executeWithBash(cmdArtemis);
+			String cmdError = queues.getError();
+			if (!Strings.isNullOrEmpty(sanitazeShellOutput(cmdError))) {
+				log.warn(
+						"[getMessagesCount] Error getting queues stats from artemis with command \"{}\" on Pod {}: \nOutput: {}\nError: {}",
+						cmdArtemis, pod.getMetadata().getName(), queues.getOutput(), cmdError);
+			}
+			try {
+				int cnt = ArtemisCliOutputParser.sumMessageCounts(queues.getOutput(), queueName);
+				messagesCount += cnt;
+			} catch (NumberFormatException | JsonSyntaxException err) {
+				log.error("[getMessagesCount] Error parsing queues stats from artemis: \n{}\n", queues.getOutput(), err);
+			}
+		}
+		return messagesCount;
+	}
+
+	/**
+	 * Counts the number of messages on every queue of every brokers' PODs
+	 * @return string containing above info formatted as a plain text message
+	 */
+	public static String getMessagesStatus() {
+		Map<String, Map<String, Integer>> podInfo = new HashMap<>();
+		// get broker pods: they are labeled with the CRD kind they were created from
+		List<Pod> pods = OpenShifts.master().getLabeledPods(ActiveMQArtemis.class.getSimpleName(), NAME);
+		for (Pod pod : pods) {
+			Map<String, Integer> queueInfo = new HashMap<>();
+			podInfo.put(pod.getMetadata().getName(), queueInfo);
+			// get queues on the broker pod
+			log.info("[getMessagesStatus] ");
+			String cmdArtemis = String.format(
+					"/home/jboss/amq-broker/bin/artemis address show --user=%s --password='%s' --url=tcp://%s:61616",
+					ADMIN_USER,
+					ADMIN_PASSWORD,
+					pod.getStatus().getPodIP());
+			PodShellOutput queues = OpenShifts.master().podShell(pod).executeWithBash(cmdArtemis);
+			String cmdError = queues.getError();
+			if (!Strings.isNullOrEmpty(sanitazeShellOutput(cmdError))) {
+				log.warn(
+						"[getMessagesStatus] Error getting queues from artemis with command \"{}\" on Pod {}: \nOutput: {}\nError: {}",
+						cmdArtemis, pod.getMetadata().getName(), queues.getOutput(), cmdError);
+			} else {
+				for (String queue : queues.getOutputAsList()) {
+					Integer count = 0;
+					if (Arrays.asList("testQueue", "inQueue", "outQueue").contains(queue)) {
```

Comment:
I know the original example uses literals too, but since those are queue names used by a specific application, I think we should use constants and add a Javadoc comment about that, e.g.: in JmsTestConstants? WDYT?